js/errors.js

```diff
@@ -217,9 +227,21 @@
      * @constructor
      * @desc An invalid filter operator
      */
-    function InvalidFilterOperatorError(message) {
+    function InvalidFilterOperatorError(message, fullURI, invalidFilter) {
```

Comment:
Also You can just pass the correct path (path without filter) to this function, since you have access to that information when you're throwing this error and you don't have to do this check again.

Reply:
We should do all the processing of data for errors outside of the error constructor. The data should be provided to the constructor and attached to the error object there.